Description: several workarounds for the build system which is completely broken
Author: Fathi Boudra <fabo@debian.org>

--- a/compat.pro
+++ b/compat.pro
@@ -66,8 +66,6 @@ TRANSLATIONS = translations/assistant_ad
     translations/assistant_adp_zh_CN.ts \
     translations/assistant_adp_zh_TW.ts
 
-unix:!contains(QT_CONFIG, zlib):LIBS += -lz
-
 contains(CONFIG, static): {
     win32 {
         exists($$[QT_INSTALL_PLUGINS]/imageformats/qjpeg.lib) {
@@ -81,3 +79,7 @@ contains(CONFIG, static): {
         }        
     }
 }
+
+target.path = $$[QT_INSTALL_BINS]
+INSTALLS += target
+
--- a/lib/lib.pro
+++ b/lib/lib.pro
@@ -16,8 +16,8 @@ HEADERS         = qassistantclient.h \
                   qassistantclient_global.h
 SOURCES         = qassistantclient.cpp
 
-DESTDIR                = ../../../../lib
-DLLDESTDIR             = ../../../../bin
+DESTDIR                = lib
+DLLDESTDIR             = bin
 
 unix {
         QMAKE_CFLAGS += $$QMAKE_CFLAGS_SHLIB
@@ -62,7 +62,7 @@ win32 {
 }
 
 qt_install_headers {
-    assistant_headers.files = $$SYNCQT.HEADER_FILES $$SYNCQT.HEADER_CLASSES
+    assistant_headers.files = qassistantclient.h qassistantclient_global.h
     assistant_headers.path = $$[QT_INSTALL_HEADERS]/QtAssistant
     INSTALLS        += assistant_headers
 }
--- /dev/null
+++ b/lib/QtAssistant/qassistantclient_global.h
@@ -0,0 +1 @@
+#include "../qassistantclient_global.h"
